src/Nav.js

```diff
@@ -69,14 +70,16 @@ function Nav({
 
       <div className="hidden md:flex items-center justify-end space-x-8 md:flex-1 lg:w-0">
         <Input
+          data-e2e-id="search-input"
+          defaultValue={searchQuery}
+          icon={searchSharp}
           id="search-bar"
           name="sound-search"
-          placeholder="Search sound..."
-          defaultValue={searchQuery}
-          data-e2e-id="search-input"
           onChange={(e) =>
             pushHistory(e.target.value) && setSearchValue(e.target.value)
           }
+          placeholder="Search sound..."
+          // type="search" // This seems appropriate, but adds an X that interferes with the search icon.
```

Comment:
Can you clarify the reason for commenting this? Not sure if I follow

Reply:
Seems like having this is the idiomatic way of implementing search:
https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types#Search_field

Reply:
[The `<Input />` component in the current version of `/src/Nav.js`](https://github.com/amilajack/tide/blob/master/src/Nav.js#L71) doesn't have a `type="search"` field. I considered adding one, since it automatically adds convenient functionality like being able to press the `Esc` key to clear the search bar, but including the `type="search"` field to the `<Input />` component adds an `X` icon that clashes with our design. So, I was wondering: should we add it anyway?

![Screen Shot 2020-12-06 at 8 43 24 PM](https://user-images.githubusercontent.com/43425812/101310420-b2c14500-3803-11eb-9683-7abfc5ac2919.png)

Reply:
The Input component shouldn't have one because we might want the component to be reusable. In the future we might want inputs that are not search inputs. I'm not able to reproduce those issues on chrome. What browser are you using?

Reply:
I used Chrome for the screenshot in my earlier comment, but the 'close' icon appears on Safari, too, if we add a `type="search"` field to the `<Input />` component in `src/Nav.js`.

Reply:
I used Chrome for the screenshot in my earlier comment, but the 'close' icon appears on Safari, too, if we add a `type="search"` field to the `<Input />` component in `src/Nav.js`.

> The Input component shouldn't have one because we might want the component to be reusable. In the future we might want inputs that are not search inputs.

I agree, which is why I only added that suggestion as [a comment to the `<Input />` component in `tide/src/Nav.js`](https://github.com/amilajack/tide/blob/bbe238de1b2ffde17f797638f6eaccb2a8279361/src/Nav.js#L82), and not [the `<Input />` component in `tide/src/components/Input.js`](https://github.com/amilajack/tide/blob/bbe238de1b2ffde17f797638f6eaccb2a8279361/src/components/Input.js#L4).

Reply:
Interesting. I'll resolve for now and we can reinvestigate later

Reply:
See https://medium.com/@rion.mrk/how-to-remove-x-icon-from-search-input-field-or-input-type-search-db3c808405fb for how to resolve this

Reply:
> See https://medium.com/@rion.mrk/how-to-remove-x-icon-from-search-input-field-or-input-type-search-db3c808405fb for how to resolve this

I revisited this and, unfortunately, neither the fix for Internet Explorer nor the fix for Chrome made the X go away in Safari. One way around this is to just have the search icon be in the beginning of the search bar like in the [Figma](https://www.figma.com/file/jeyLWmyn78DjqSmbWcJOEd/Tide?node-id=798%3A84), instead of at the end of the search bar like it is currently.